scylla-rust-wrapper/src/cluster.rs

```diff
@@ -49,6 +50,11 @@ impl LoadBalancingConfig {
     // as it results in panic due to DefaultPolicyBuilder::build() spawning a tokio task.
     pub(crate) async fn build(self) -> Arc<dyn LoadBalancingPolicy> {
         let mut builder = DefaultPolicyBuilder::new().token_aware(self.token_awareness_enabled);
+        if self.token_awareness_enabled {
```

Comment:
I would recommend to add another option to enable shuffling no matter what:
```suggestion
        if self.shuffling_replicas_enabled || (self.token_awareness_enabled && self.token_aware_shuffling_replicas_enabled) {
```